tests/tiup-dm/test_cmd.sh

```diff
@@ -69,6 +69,18 @@ topo_worker=./topo/full_scale_in_dm-worker.yaml
 sed "s/__IPPREFIX__/$ipprefix/g" $topo_worker.tpl > $topo_worker
 yes | tiup-dm scale-out $name $topo_worker
 
+echo "start scale in grafana"
+yes | tiup-dm scale-in $name -N $ipprefix.101:3000
+wait_instance_num_reach $name $total_sub_one
+
+echo "start scale out grafana"
+topo_grafana=./topo/full_scale_in_grafana.yaml
+sed "s/__IPPREFIX__/$ipprefix/g" $topo_grafana.tpl > $topo_grafana
+yes | tiup-dm scale-out $name $topo_grafana
+
+# test grafana config
+tiup-dm exec $name -N $ipprefix.101 --command "ls /home/tidb/deploy/grafana-3000/dashboards/dm.json && ! grep magic-string-for-test /home/tidb/deploy/grafana-3000/dashboards/dm.json"
```

Comment:
Should `! grep` be the opposite? Curious abot the test-case was passed, but this file contains this string 
https://github.com/pingcap/tiup/blob/7c52950c0bf42e664777a6839d60af6ad5e33dca/tests/tiup-dm/local/grafana/dm.json#L177

Reply:
Because in the scale-out topo ($topo_grafana), we didn't specify dashboard_dir, so it will use the built-in one which doesn't contains the magic string.